docs/source/Contribute.md

```diff
@@ -98,8 +98,8 @@ $ flake8 . --show-source --statistics
 2. Run the build and start the app using the docker-compose test file. In this way, you would launch the code in your environment and not the last official image in Docker Hub:
 
 ```bash
-$ docker-compose -f docker-compose-for-travis.yml build
-$ docker-compose -f docker-compose-for-travis.yml up
+$ docker-compose -f docker/default.yml -f docker/travis.yml build
```

Comment:
shouldn't it be ~`docker/travis.yml build`~ `docker/travis.override.yml` ?

Reply:
same for line 102

Reply:
Right again